### ISSUE TITLE
how to install noghost?

### ISSUE THREAD

**Issue by mastersam2015** [inolen/quakejs#60]
how to install noghost?

**Comment by briancullinan** [comment]
@mastersam2015 Start by asking the original author to post source-code to the qagame files.

**Comment by fireflypie** [comment]
Yeah.. just creating a folder eg, noghost/ and running it with node build/ioq3ded.js +set fs_game noghost +set dedicated 1 +exec server.cfg
 won't work...
